task4-kernel-module-hello/Makefile

```diff
@@ -0,0 +1,22 @@
+ifneq ($(KERNELRELEASE),)
+	obj-m := hello.o
+else 
+	KDIR ?= /lib/modules/`uname -r`/build
+
+INS=hello #Write your name of module without extension
+
```

Comment:
Just an advice: As I can understand here, 'hello' is a module name.
If there is an intent to use some environment variable to store the name, so 
1. I'd suggest to use some meaningful name, so people will save his time guessing what does "INS" mean and used for.
2. It should be used everywhere, so I'd suggest to move declaration to the top of the files and use variable in the `obj-m := hello.o` line.